services/frontend/src/components/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,78 @@
+import React, {useMemo} from "react";
+import classNames from "classnames";
+import {Card, CardContent, IconButton} from "@material-ui/core";
+import {ChevronRight} from "@material-ui/icons";
+import "./ProjectCard.scss";
+
+const DISPLAY_SKILL_COUNT = 3;
+
+const ProjectCard = ({className, name, description, skills, isActive}) => {
+    // Only show the first DISPLAY_SKILL_COUNT skills, so as to not crowd the card too much
+    const skillBadges = useMemo(() => skills.slice(0, DISPLAY_SKILL_COUNT).map((name) => (
+        <SkillBadge
+            key={name}
+            name={name}
+        />
+    )), [skills]);
+
+    return (
+        <Card className={classNames("project-card", className)}>
+            <CardContent className="project-card-content">
+                <div className="project-card-activity-section">
+                    <ActiveBadge isActive={isActive} />
+                </div>
+
+                <div className="project-card-content-section">
+                    <h3 className="project-card-name">{name}</h3>
+                    <p className="project-card-description">{description}</p>
+
+                    <div className="project-card-skills">
```

Comment:
The cards are better this way (splitting `project-card-skills` and `project-card-more-skills` however it's causing a css issue in the projects page.

before you had:
```
<div className="project-card-skills">
    {skillBadges}

    {skills.length > DISPLAY_SKILL_COUNT && (
        <div className="project-card-more-skills">
            More skills...
        </div>
    )}
</div>
```

Which was extending the skills out so that the card was larger than your `max-width` property. The cards right now are different sizes since they are too small for `max-width`.

Reply:
The code didn't effectively show where the problem is, my bad. It's from line 29-38

Reply:
Good catch! I've changed it now so that the cards all get a fixed width by default, but can start shrinking when the screen gets too small for the size of the card.